### ISSUE TITLE
Connecting to posefs1.perception.cs.cmu.edu failed: Unknown error

### ISSUE THREAD

**Issue by koljanich** [CMU-Perceptual-Computing-Lab/openpose_unity_plugin#90]
Hello! When I try to load models (getModels.bat) I get this error: "Connecting to posefs1.perception.cs.cmu.edu (posefs1.perception.cs.cmu.edu)|128.2.220.57|:80... failed: Unknown error. Retrying."

**Comment by aditdesai** [comment]
Did you find a solution?

**Comment by legendarystar143590** [comment]
Please use this link to download the models from google drive.
https://drive.google.com/drive/folders/1USEdy_7uvwO4PIqsQJq8kT0sX4H4f7nn

**Comment by JIshanRandika** [comment]
I had same issue. I solved with updating the models using above google drive link.
Thank you @legendarystar143590 !

**Comment by Vuoti** [comment]
is there a new link to download the models? the google drive link broken as well

**Comment by ElliotEserinUT** [comment]
@Vuoti Did you find a new location to download the models from?

**Comment by Vuoti** [comment]
Yeah I think I found them in the openpose main project / another repo
